transactor/README.md

```diff
@@ -24,26 +43,50 @@ val chunkTableAttrs = List(new AttributeDefinition("chunkId", "S"))
 val chunkTableSchema = List(new KeySchemaElement("chunkId", KeyType.HASH))
 dynamo.createTable(chunkTableAttrs, "TestChunks", chunkTableSchema, pth)
 ```
-* Create config file
+
+## Run a transactor server
+To run a transactor server:
 ```
-io.mediachain.transactor.server.rootdir: /path/to/transactor-directory
-io.mediachain.transactor.server.address: 127.0.0.1:10000
-io.mediachain.transactor.dynamo.awscreds.access: $AWSACCESS
-io.mediachain.transactor.dynamo.awscreds.secret: $AWSSECRET
-io.mediachain.transactor.dynamo.baseTable: Test
-io.mediachain.transactor.dynamo.endpoint: http://localhost:8000
+java -cp $BIGJAR io.mediachain.transactor.Main transactor -c <config-file> cluster-address ...
 ```
 
-If you want to enable SSL, add the following to the config file:
+To run the RPC facade:
+```
+java -cp $BIGJAR io.mediachain.transactor.Main facade -c <config-file> cluster-address ...
+```
+
+Both servers can be shutdown by signalling a kill file. The control
+directory is set in the configuration file; the transactor control directory
+is at `${io.mediachain.transactor.server.rootdir}/ctl`, while the rpc service
+control directory is `io.mediachain.transactor.rpc.control`.
+
+### transactor configuration
+The configuration is a properties file.
+
+Required properties:
+```
+io.mediachain.transactor.server.rootdir: /path/to/transactor-dir # Server root directory
+io.mediachain.transactor.server.address: 127.0.0.1:10000         # Server bind address
+io.mediachain.transactor.dynamo.baseTable: Test                  # DynamoDB Table
+io.mediachain.transactor.dynamo.endpoint: http://localhost:8000  # Optional
+```
+
+Optionally, if you want to enable SSL, add the following to the config file:
 ```
 io.mediachain.transactor.ssl.enabled: true
 io.mediachain.transactor.ssl.keyStorePath: /path/to/keystore.jks
 io.mediachain.transactor.ssl.keyStorePassword: $KEYSTOREPASS
 io.mediachain.transactor.ssl.keyStoreKeyPassword: $KEYSTOREKEYPASS
 ```
 
-* `sbt transactor/assembly`
-* `scala -cp $BIG_JAR_FROM_ABOVE io.mediachain.transactor.JournalServer path/to/config.conf`
+### RPC facade configuration
+Required properties:
+```
+io.mediachain.transactor.rpc.port: 10001
+io.mediachain.transactor.rpc.control: /path/to/rpc/killfile-dir
```

Comment:
The dynamo access was added by the streaming branch, but the README was not updated.